packages/go_router/test/go_router_test.dart

```diff
@@ -1012,6 +1012,80 @@ void main() {
       ]);
     });
 
+    testWidgets('Can manually pop root navigator and display correct url',
```

Comment:
Can we also add a test case that pop multiple times?
I think this PR would fix https://github.com/flutter/flutter/issues/117431 but just want to make sure of it.

I added a test case for it in this PR: https://github.com/flutter/packages/pull/2989 but since it's close, I'd like to add it in this PR at least.

Reply:
added `on pop twice`

Reply:
thanks ;)